### ISSUE TITLE
Validace snippetů kódu

### ISSUE THREAD

**Issue by FilipJirsak** [podlomar/kodim.cz#65]
Relativně často se stává, že v příkladech v kódu jsou syntaktické chyby. Bylo by fajn, kdyby bylo možné vložený kód nějak testovat alespoň na syntaktickou správnost.

Líbilo by se mi, kdyby bylo možné odkazovat z Markdownu na kód v externích souborech. Ještě lepší by bylo, kdyby šlo odkázat jenom na vybrané řádky (a některé řádky třeba umět ještě zvlášť zvýraznit, třeba takhle: https://codehike.org/demo/code). Pak by bylo možné mít příklady kódu v externím souboru třeba i s testem nebo řešením a tyhle externí soubory testovat třeba jednotkovými testy.

**Comment by FilipChalupa** [comment]
> Relativně často se stává, že v příkladech v kódu jsou syntaktické chyby. Bylo by fajn, kdyby bylo možné vložený kód nějak testovat alespoň na syntaktickou správnost.

Tohle je podle mě odpovědnost repozitáře s materiály. Podobně, jako se teď třeba testuje akademie prettierem. (Co by mělo udělat kódím, když narazí na syntax error, ale pořád je schopné ten kód zobrazit?)

Jinak zrovna v akademii je několikrát v bloku kód záměrně chybně jako ukázka, jak to nemá být. Nejsem toho absolutně fanoušek, ale i to by se muselo možná dořešit. I když vlastně z pohledu syntaxe ty kódy asi budou všechny správně (`if (7 > x > 2)` nebo `document.kverySelektor('div')`).

**Comment by FilipJirsak** [comment]
Souhlasím, řešit by to měl repositář s materiály. Nicméně by to chtělo nějakou společnou podporu, což je nějaký kód – asi nový projekt. Sem jsem to dal jenom proto, že do konkrétního repository s materiály to nepatří vůbec :-)
Co se týče samotné validace, určitě by to muselo být pro konkrétní blok kódu vypínatelné.
